packages/main/test/pages/SegmentedButton.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```

Comment:
In addition you have to create Segmented.sample.html page for the playground, use for reference the Button.sample.html or ToggleButton.sample.html